f/medical/f_CLSEH.sqf

```diff
@@ -0,0 +1,39 @@
+// FA3 - Combat Life Saver Event Handler
+// This component adds an event handler for use with the Combat Life Saver assignGear class. Units with the f3_cls trait can provide full heals using FAKs.
+// Enable this component in init.sqf
+// For credits and more information see https://github.com/folkarps/F3/wiki
+
+// Make sure the player can be broadcast properly
+_unit = player;
+
+// Add the EH
+[_unit,	["HandleHeal", {
+		_this spawn {
+            params ["_injured", "_healer","_isMedic"];
+				
+
+			
+				// Check whether the person healing the player is a CLS
+                if (_healer getUnitTrait "f3_cls") then {
+				
+					// DEBUG
+					if (f_param_debugMode == 1) then
+						{
+							player sideChat format ["DEBUG (f_CLSEH.sqf): %1 is CLS healing %2",(name _healer),(name _injured)];
+						};
+						
+					// Wait until the standard heal has been applied
+					waitUntil {damage _injured <= 0.25};
```

Comment:
Might want to have a timeout here. Probably doesn't matter but you could spawn a bunch of threads by repeatedly failing heals.

Also maybe change the number to .2501 or something? .25 is base 2 so it proooobably doesn't matter, but number comparisons are scary.

Reply:
All sorted. Probably.